teammapper-backend/src/map/utils/clientServerMapping.ts

```diff
@@ -95,7 +95,7 @@ const mapClientBasicNodeToMmpRootNode = (
   imageSrc: clientRootNodeBasics.image?.src,
   imageSize: clientRootNodeBasics.image?.size,
   name: clientRootNodeBasics.name || DEFAULT_NAME,
-  nodeParentId: null,
+  nodeParentId: undefined,
```

Comment:
I think we can remove this line, as a missing key should be undefined in the entity

Reply:
Yeah, removed.